amoro-ams/amoro-ams-server/src/main/java/org/apache/amoro/server/terminal/kyuubi/KyuubiTerminalSessionFactory.java

```diff
@@ -124,8 +125,11 @@ public TerminalSession create(TableMetaStore metaStore, Configurations configura
     Properties properties = new Properties();
 
     if (!metaStore.isKerberosAuthMethod()) {
-      properties.put(JdbcConnectionParams.AUTH_USER, metaStore.getHadoopUsername());
```

Comment:
How about checking if the value is null before putting it into properties and configuration?
I think it will be simpler if we fix it this way.

Reply:
+1

Reply:
Thank you for your suggestion, I have made the changes.

Reply:
We can use one condition to fix this, WDYT

Reply:
> We can use one condition to fix this, WDYT

Okay, I've merged it into one.thinks.